### PR TITLE
feature(tpi: testimonials): add testimonials scaffold properly styled…

### PR DESCRIPTION
**Pull request by rapzo**
… as spec'd

### CODE REVIEW

File: app/views/tpi/home/_testimonials.html.erb [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
-      <p class="testimonials__content">
-        Businesses should be able explain to investors how they plan to manage climate change risks, invest and innovate on the way to the zero-carbon economy of the future. With the launch of the Transition Pathway Initiative, asset owners from around the world are sending a strong signal that portfolios will align in the future with companies that are taking the transition to a low carbon economy seriously.
-      </p>
-
-      <p class="testimonials__author">
-        Emma Howard Boyd
-        Chair of the Environment Agency
-      </p>
-    </div>
+    <div class="column is-10 is-offset-1 testimonials">
```
Comment (aabdaab) [inline app/views/tpi/home/_testimonials.html.erb]: once you're gonna start adding the behavior to it you might end up moving everything to `TestimonialSlider` component, I think it's gonna be easier than exposing only some functions from the component, don't even know how that would work 🤔

File: app/views/tpi/home/_testimonials.html.erb [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
+        <div class="column is-2 testimonials__quote">
+          <img src="<%= asset_path 'quote.svg' %>" alt="">
+        </div>
+        <div class="column is-8">
+          <figure class="image is-92x92">
+            <img src="<%= asset_path 'clint_eastwood.jpg' %>" class="is-rounded testimonials__avatar" alt="">
+          </figure>
+
+          <div class="testimonials__content">
+            <p>You see in this world there are two kinds of people my friend. Those with loaded guns, and those who dig. You dig.</p>
```
Comment (aabdaab) [inline app/views/tpi/home/_testimonials.html.erb]: no, you dig 🔫

File: app/views/tpi/home/_testimonials.html.erb (hunk 1 above)
Comment (rapzo) [reply app/views/tpi/home/_testimonials.html.erb]: Yeah, that's the idea (the other testimonials are not even updated as the 1st one, they moer liek placeholders)...
Right now i'm adding dummy data like:
```
[{
  author: 'Lee Van Cleef',
  role: 'Golden Boot Award Winner',
  content: ['fooo', 'baaar', 'baaaz'],
  avatar_url: 'xxx.jpg',
}]
```

But i have mixed feelings if it is worth throwing react into this...

What do you suggest?

File: app/views/tpi/home/_testimonials.html.erb (hunk 2 above)
Comment (rapzo) [reply app/views/tpi/home/_testimonials.html.erb]: Blondie was such a badass 🤣

File: app/views/tpi/home/_testimonials.html.erb (hunk 1 above)
Comment (aabdaab) [reply app/views/tpi/home/_testimonials.html.erb]: I think it's gonna be more natural with React but as you prefer! what did you have in mind in the beginning?

Comment (rapzo) [reply app/views/tpi/home/_testimonials.html.erb]: a bunch of `addEventListener`s and `querySelector`s replacing `innerHTML`s...

**Comment by rapzo** [comment]
Erm.. Can't make the PR WIP now, i'm wrapping the fade-in/fade-out animation that Fausto requested.
I'll drop here a comment once done. Please don't accept this meanwhile.

**Comment by tsubik** [comment]
Ok!

**Comment by tsubik** [comment]
Not sure if testimonials have to be manageable by admin? If yes we need to add story for that @simaob

**Comment by rapzo** [comment]
Stashed all changes related to the fade-in/fade-out since i was losing too much time trying to sync the 8 second transition between quotes, plus the timeout to start the fade-in and timeout to fade-out, plus halting everything when a quote is clicked and reseting timers on quote page select.
Will get to this again later.

**Pull request merged**